Makefile

```diff
@@ -510,6 +510,18 @@ out/doc/api/%.html: doc/api/%.md
 		fi
 	[ -x $(NODE) ] && $(NODE) $(gen-html) || node $(gen-html)
 
+# check if ./node is actually set, else use user pre-installed binary
+gen-man = tools/doc/generate.js --format=man $< > $@
+out/doc/api/%.3: doc/api/%.md
+	@[ -e tools/doc/node_modules/js-yaml/package.json ] || \
+		[ -e tools/eslint/node_modules/js-yaml/package.json ] || \
```

Comment:
I copied this from the HTML build logic, but it's now duplicated three times. Should we refactor this? I think yes, but my Make-fu is not very good...